### PR TITLE
Added a bunch of tags, and organized some stuff

### PR DESCRIPTION
**Pull request by njaremko**

### CODE REVIEW

**Comment by egtann** [comment]
Some notes: the comments use Golang's documentation, so they're visible on godoc.org and using Golang's doc generation tools. They have to follow a specific format; they must start with the function name.

File: shared/language/language.go [hunk 1]
```diff
@@ -8,7 +8,9 @@
 //	3. Consolidates triggers by categories (e.g. automotive brands) if
 //	commonly used across plugins.
 //	4. Summarizes text using the custom rule-based algorithm found in
-//	summarize.go.
+//	summarize.go
+//
+//  Very useful for object triggers
```
Comment (egtann) [inline shared/language/language.go]: This should be written in a complete sentence following the style of the other comments.

File: shared/language/language.go [hunk 2]
```diff
@@ -172,117 +174,176 @@ func SuggestedProduct(s string, num uint) string {
...
+		"muffin",
+		"pie",
+		"shortbread",
+		"shortcake",
+		"square",
+		"tart",
+	}
+}
+
+// Returns a []string of restaurant types useful in a plugin's object
```
Comment (egtann) [inline shared/language/language.go]: First word should be Restaurants

**Comment by egtann** [comment]
This looks ready to merge as well. Let's just rebase and force-push into a single commit.

**Comment by njaremko** [comment]
Don't merge this yet, I had so many wonderful types of alcohol in Alcohol().... :( On the wrong computer I guess...

**Comment by njaremko** [comment]
Okay, I'm satisfied. :+1:

**Comment by egtann** [comment]
Merged, thanks!

**Pull request merged**